packages/openapi-ts/src/plugins/@tanstack/query-core/plugin.ts

```diff
@@ -628,8 +628,8 @@ const createQueryKeyLiteral = ({
 }: {
   isInfinite?: boolean;
   operation: Operation;
-}) => {
-  const queryKeyLiteral = compiler.arrayLiteralExpression({
+}) =>
+  compiler.arrayLiteralExpression({
```

Comment:
I assume this change was done while you were playing around with the code, mind reverting it?

Reply:
IntellIiJ complained, because of the redundant variable, I though I will just fix it. But of course I can revert it.